### ISSUE TITLE
Ability to use a different timezone?

### ISSUE THREAD

**Issue by ashishb** [olebedev/when#23]
I couldn't see to find any timezone parsing code. So, for example, if I am saying "10 PM EST" while the machine's local time is in PST then I expect 7 PM as the outcome in local time or 10 PM with the timezone information extracted.

**Comment by olebedev** [comment]
It's supposed that you already know or parse a timezone and give a `time.Time` instance with already set up timezone in it.

**Comment by ashishb** [comment]
I want to extract it from the free-form text.

**Comment by ShadiestGoat** [comment]
I think this issue should be re-opened. It should be able to parse "7pm PST" and "7pm UTC" differently.

Thank you!

**Comment by olebedev** [comment]
Yeah, it seems like a useful feature to have in the library. I think it can be relatively easy implemented as a rule, that parses only timezones if any and adjust the result accordingly.

I open the issue to keep it visible. It would be really cool if someone can spend a couple of hours implementing the rule :) I will be happy to review and merge when its ready to.
